Vostok.ClusterConfig.Core/Parsers/FileParserSettings.cs

```diff
@@ -24,5 +24,17 @@ internal class FileParserSettings
         };
 
         public int MaximumFileSize { get; set; } = 1024 * 1024;
+
+        public Dictionary<string, long> MaximumFileSizeZoneOverloads { get; set; } = new Dictionary<string, long>
```

Comment:
Zone names are case-insensitive in ClusterConfig (so we probably should use a case-insensitive comparer here).